charts/partials/templates/_proxy.tpl

```diff
@@ -168,6 +171,15 @@ readinessProbe:
     path: /ready
     port: {{.Values.proxy.ports.admin}}
   initialDelaySeconds: 2
+{{- if and .Values.proxy.nativeSidecar .Values.proxy.await }}
+startupProbe:
+  httpGet:
+    path: /ready
+    port: {{.Values.proxy.ports.admin}}
+  initialDelaySeconds: {{.Values.proxy.startupProbeInitialDelaySeconds | default 0}}
```

Comment:
I think these variables should be added to values.yaml.  This will allow us to set default values in values.yaml instead of needing to do defaulting inline in the template like this.

Reply:
The inline defaults are for injection because I didn't plumb the values into `pkg/charts/linkerd2.Proxy`.

I'm happy to do this, but I think this would expose those settings to users to modify which we discussed above as being out of scope or perhaps undesired.  Please let me know if I'm mistaken.